### ISSUE TITLE
Where is the Pascal Context evaluation server??

### ISSUE THREAD

**Issue by CoarseSand** [zhanghang1989/PyTorch-Encoding#257]
Hello,

I know this must sound like a stupid question,
but I couldn't find information about the **evaluation server for the Pascal Context dataset.**
I'm assuming since the Pascal Context dataset is an extension of Pascal VOC 2010, that I should submit my results on the test set to the Pascal VOC 2010, but I am not too sure. Could you please tell me where I can submit the results on the test set to?

Thank you in advance :)

**Comment by zhanghang1989** [comment]
You may use the provided validation set and evaluate locally, see instructions at https://hangzhang.org/PyTorch-Encoding/model_zoo/segmentation.html#test-pretrained

I don't think they have server for test set.

**Comment by CoarseSand** [comment]
Thank you for replying so promptly to this issue.
I think there must be some kind of test server for the Pascal Context because the following papers cite "Pascal-context test set results" when comparing against state of the art:

1. Expectation-Maximization Attention Networks for Semantic Segmentation (Li et. al, 2019)
2. Dual Attention Network for Scene Segmentation (Fu et. al, 2018)

**Comment by zhanghang1989** [comment]
Maybe they recently set up one. I didn't follow recent work in Semantic Segmentation.

**Comment by zhanghang1989** [comment]
I am closing it, since it is not issue for this repo.

**Comment by zhanghang1989** [comment]
Hi @CoarseSand , have you figure out how to submit to Pascal Context test server?

**Comment by CoarseSand** [comment]
Hello @zhanghang1989, I'm afraid I have not :(( although I have noticed that what many people refer to as the "test set" appears to be in fact, the validation set which 1000 images that comes along as part of the pascal context dataset. So I plan to evaluate on the test set. My problem I have now is that I am having trouble tailoring the multiscale evaluation module to my target reference. I would really appreciate it if you could reply to issue #261 :)) 

Thanks in advance

**Comment by zhanghang1989** [comment]
I have also emailed to the author of a paper reporting the score on test set. He confirmed that there is no server for test set, but they evaluate the performance locally since they have the access to test set label.
